packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx

```diff
@@ -385,17 +402,35 @@ function MapAnalysisImpl(props: ImplProps) {
         overlayEntity,
         dataClient,
         subsettingClient,
-        binningMethod: activeMarkerConfiguration?.binningMethod,
+        markerType: activeMarkerConfiguration?.type,
+        binningMethod:
+          activeMarkerConfiguration &&
+          'binningMethod' in activeMarkerConfiguration
+            ? activeMarkerConfiguration.binningMethod
+            : undefined,
+        aggregator:
+          activeMarkerConfiguration && 'aggregator' in activeMarkerConfiguration
+            ? activeMarkerConfiguration.aggregator
+            : undefined,
+        numeratorValues:
+          activeMarkerConfiguration &&
+          'numeratorValues' in activeMarkerConfiguration
+            ? activeMarkerConfiguration.numeratorValues
+            : undefined,
+        denominatorValues:
+          activeMarkerConfiguration &&
+          'denominatorValues' in activeMarkerConfiguration
+            ? activeMarkerConfiguration.denominatorValues
+            : undefined,
```

Comment:
I do this sort of conditional property accession quite a few times. Anyone know of more succinct way to do this that both grabs the value and narrows the type? For instance, Lodash's `get` function is a great way to grab the value conditionally, but it doesn't narrow the type.

Reply:
Okay so technically `_.get()` does work for this particular example, but there are a couple of others like this that it doesn't work for, because TS thinks the expression can only lead to a value of `undefined`